src/DisputeHandler.ts

```diff
@@ -53,7 +46,7 @@ class DisputeHandler {
         this.agreementManager = agreementManager;
         this.stateChannelManagerContract = stateChannelManagerContract;
         this.p2pEventHooks = p2pEventHooks;
-        return this.self;
```

Comment:
Should keep this for debugging - if the debugging flag is set to true the constructor will return the proxy object useful for debugging otherwise it returns the normal this

Reply:
restored